### PR TITLE
Fixes #8380 : Fix flaky test in elasticsearch: Change aliasName to be unique

### LINKED ISSUE #8380: Fix flaky test `testAlias()` in BaseElasticsearchConnectorTest
Test `testSelectInformationSchemaColumns()` asserts query
https://github.com/trinodb/trino/blob/ea5d9889b0844e83ca90673fff529632931e2561/testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java#L664-L666.

But another test `testAlias()` creates alias by name `orders_alias` which runs always after the latter test. If `testAlias()` runs before the latter then the above assert fails.

So to ensure that tests remain independent of each other, aliasName in `testAlias()` should be renamed.

### PR DESCRIPTION
**Pull request by sumannewton**
Fixes #8380

### CODE REVIEW

File: plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java [hunk 1]
```diff
@@ -1076,10 +1077,11 @@ public void testQueryStringError()
     public void testAlias()
             throws IOException
     {
-        addAlias("orders", "orders_alias");
+        String aliasName = format("alias_%s", Math.abs(UUID.randomUUID().hashCode()));
```
Comment (hashhar) [inline plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java:1080]: can we use `TestTable.randomTableSuffix()` here?

Comment (sumannewton) [reply plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java]: Done

**Comment by sumannewton** [comment]
@martint @findepi Requesting your review. Thanks.

**Review by phd3 — APPROVED** [review]
Thanks! Could you update commit message to something like below? 

```
Fix flaky testSelectInformationSchemaForMultiIndexAlias

Rename table alias in testAlias to avoid accidentally matching 
regular expression for information schema queries in 
testSelectInformationSchemaForMultiIndexAlias
```

FWIW, IMO it's easy to mess with `information_schema` tests (especially for regexes) while writing new tests. we can think about making those tests independent by having stricter regex patterns, may be with random suffixes. But for now, this looks good.

File: plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java [hunk 2]
```diff
@@ -1076,10 +1077,11 @@ public void testQueryStringError()
     public void testAlias()
             throws IOException
     {
-        addAlias("orders", "orders_alias");
+        String aliasName = format("alias_%s", TestTable.randomTableSuffix());
```
[the remark this answers is not in the document]
Comment (sumannewton) [reply plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java]: Done.

**Comment by sumannewton** [comment]
> Thanks! Could you update commit message to something like below?

Done.

**Review by losipiuk — APPROVED** [review]

**Comment by losipiuk** [comment]
I moved the fix commit to https://github.com/trinodb/trino/pull/8379. So we can merge reeanbling tests and fix together.

**Pull request closed without merging**